$NetBSD: patch-ag,v 1.1 2006/09/21 21:47:32 dsainty Exp $

Fix build under gcc 4.1.2, which catches an extern Vs static inconsistency.

--- src/VideoUla.h.orig	2002-01-16 04:46:43.000000000 +1300
+++ src/VideoUla.h	2006-09-22 01:06:05.000000000 +1200
@@ -88,6 +88,5 @@
 extern	unsigned char		CursorByteWidth;
 extern	unsigned char		MasterCursorWidth;
 extern	unsigned char		ClockRate;
-extern	byteval				VidRegister0;
 
 #endif	/* VIDEOULA_H */
